Doc/whatsnew/3.14.rst

```diff
@@ -715,6 +716,33 @@ in the :envvar:`PYTHONSTARTUP` script.
 (Contributed by Łukasz Langa in :gh:`131507`.)
 
 
+.. _whatsnew314-jit-compiler:
+
+An experimental just-in-time compiler
+-------------------------------------
```

Comment:
I think this is a clearer headline, especially as 3.13 included the same item. Weakly-held, though.
```suggestion
Binary releases for the experimental just-in-time compiler
----------------------------------------------------------
```